packages/tools/src/tools/annotation/EllipticalROITool.ts

```diff
@@ -125,6 +126,9 @@ class EllipticalROITool extends AnnotationTool {
       configuration: {
         shadow: true,
         preventHandleOutsideImage: false,
+        centerPointRadius: 0, // Radius of the circle to draw  at the center
```

Comment:
can you please update the class comment above to document this so that it appears in the docs here https://www.cornerstonejs.org/api/tools/class/ellipticalroitool/